main.py

```diff
@@ -1,7 +1,24 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name = "Western",)
+    Genre.objects.create(name="Action",)
+    Genre.objects.create(name="Dramma")
+    Actor.objects.create(first_name = "George", last_name = "Klooney")
+    Actor.objects.create(first_name="Kianu", last_name="Reaves")
+    Actor.objects.create(first_name="Scarlett", last_name="Keegan")
+    Actor.objects.create(first_name="Will", last_name="Smith")
+    Actor.objects.create(first_name="Jaden", last_name="Smith")
+    Actor.objects.create(first_name="Scarlett", last_name="Johansson")
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name = "George",
+                         last_name = "Klooney").update(last_name = "Clooney")
```

Comment:
There seems to be a redundant update operation for 'George Klooney'. Ensure that the intended update is performed correctly and efficiently.